src/migrate.js

```diff
@@ -1,18 +1,34 @@
 
-var warnedAbout = {};
+( function() {
 
-// List of warnings already given; public read only
-jQuery.migrateWarnings = [];
+	// Support: IE9 only
+	// Console can go away during the page run so don't cache globally
+	var log = window.console && window.console.log.bind( window.console ),
+		rbadVersions = /^[12]\./;
 
-// Set to true to prevent console output; migrateWarnings still maintained
-// jQuery.migrateMute = false;
+	if ( !log ) {
+		return;
+	}
+
+	// Need jQuery 3.0.0+ and no older Migrate loaded
+	if ( !window.jQuery || rbadVersions.test( jQuery.fn.jquery ) ) {
```

Comment:
Drop the `window.` prefix; `jQuery` is already a [local variable](https://github.com/dmethvin/jquery-migrate/blob/115-only3/src/intro.js).